docker.local/config/b0magicBlock_3_miners_3_sharders.json

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
Do we have any changes that require the rebuilding of magic block, if yes, I'll need to rebuild it on hero sever.

Reply:
I think, that it was some experiment, probably we can remove it from commit, this file is not necessary